addons/website_slides/models/slide_channel_tag.py

```diff
@@ -30,3 +30,4 @@ class SlideChannelTag(models.Model):
         'Group sequence', related='group_id.sequence',
         index=True, readonly=True, store=True)
     channel_ids = fields.Many2many('slide.channel', 'slide_channel_tag_rel', 'tag_id', 'channel_id', string='Channels')
+    color = fields.Integer(string='Color Index', help="Color to apply to this tag (including in website).")
```

Comment:
You will have to write a small migration script to create the column for this one. Not all tech gooroos seem to agree but currently migration team ask for it :) .

Reply:
Done. I only added the migration PR link in the commit related to this. Hope that's correct.